sdk/eventhub/azure-eventhub/samples/async_samples/sample_code_eventhub_async.py

```diff
@@ -113,10 +113,13 @@ def example_create_async_eventhub_consumer_client():
 
         async def on_event(partition_context, event):
             logger.info("Received event from partition: {}".format(partition_context.partition_id))
-            # Do asnchronous ops on received events
+            # Do some asynchronous ops on received event
```

Comment:
Minor suggestion: keep it consistent with the sample codes

```python
        async def on_event(partition_context, event):
            # Put your code here.
            # If the operation is i/o intensive, async will have better performance.
            logger.info("Received event from partition: {}".format(partition_context.partition_id))
```

Reply:
Adopted your suggestion